volatility3/framework/plugins/windows/driver_signing_enforcement.py

```diff
@@ -0,0 +1,124 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+import logging
+from typing import List
```

Comment:
Thank you for your comment @ikelos,
That's right, I think I forgot to present the necessary modules as I used the function to suggest a better way.
If the proposal below is included, I think it will be more stable.

```suggestion
from typing import Iterator, List, Optional
```